WalletWasabi.Gui/Tabs/WalletManager/GenerateWallets/GenerateWalletSuccessView.xaml

```diff
@@ -3,13 +3,27 @@
              xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui"
              x:Class="WalletWasabi.Gui.Tabs.WalletManager.GenerateWallets.GenerateWalletSuccessView">
   <StackPanel Margin="20" Spacing="10">
-    <TextBlock Text="Write down these Recovery Words!" FontWeight="Bold" />
-    <Border Padding="2" Background="White" HorizontalAlignment="Stretch" Opacity="0.8">
-      <controls:ExtendedTextBox Text="{Binding MnemonicWords, Mode=TwoWay}" Background="Transparent" FontWeight="Bold" Foreground="Black" HorizontalAlignment="Center" Classes="selectableTextBlock" />
+    <TextBlock Text="Write down these Recovery Words &amp; Your Password" FontWeight="Bold" />
+    <Border Padding="20" Margin="20 0" HorizontalAlignment="Center">
+      <ItemsControl Items="{Binding MnemonicWords}">
+        <ItemsControl.Styles>
+          <Style Selector="TextBlock">            
+            <Setter Property="FontSize" Value="22" />
+            <Setter Property="Margin" Value="40 10" />
+            <Setter Property="Height" Value="32" />
+          </Style>
+        </ItemsControl.Styles>
+        <ItemsControl.ItemsPanel>
+          <ItemsPanelTemplate>
+            <WrapPanel MaxHeight="220" Orientation="Vertical" />
+          </ItemsPanelTemplate>
+        </ItemsControl.ItemsPanel>
+      </ItemsControl>
     </Border>
     <TextBlock Text="You can recover your wallet on any computer with" />
     <TextBlock Text="- your Recovery Words AND" />
```

Comment:
How about this?
```suggestion
    <TextBlock Text="Your wallet can only be recovered with both:" />
    <TextBlock Text="- your Recovery Words." />
```

Reply:
im keeping wording the same as current version its been back and forth too many times.

Reply:
That `AND` shouldn't be on the same line of the first item, it doesn't look good.